app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2>Sign up</h2>
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
```

Comment:
[OPTIONAL] It would be excellent if the user was automatically logged in or redirected to the login page after providing the information and clicking the sign-up button. Your APP displays the following error instead:

<img width="658" alt="image" src="https://user-images.githubusercontent.com/77190203/175332433-b0ad5188-3b00-4466-908b-1a375c3125bd.png">

Reply:
This is interesting. I don't know why we didn't see it before.
Will check how to handle that.

Reply:
@medaminedev66 I am solving this by making a redirect from user_url to root_path. Is there any better way? Is there any hoop in devise with which we can change the redirect targer after a given action (in this case sign_up)? I tried to find something but I couldn't

Reply:
I would like to learn more. So I like such optional changes when there is some good reason behind them.

Thanks you so much for your thorough review. After very long time I am getting such a good reviews. :+1: